### PR TITLE
Fix login styling

### PR DESCRIPTION
**Pull request by Rashmil-1999**
This PR will fix login styling issue.

### CODE REVIEW

**Comment by Rashmil-1999** [comment]
@Vismayak @longshuicy

File: src/components/childComponents/graph/SectorShocksBarChart.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { csvToArray } from "../../../utils/common";
 const ParseData = (data) => {
 	const chartData = [];
 	const chartDataObj = {};
-	const colors = ["#aededb","#2d8686","#00619D","#2c698d","#272643"];
+	const colors = ["#DB4F60", "#ff7e0f", "#A1BE2E", "#1e77b4", "#3fbecf"];
 	chartData.push(["Sector", "Ratio of Capital Remaining", { role: "style" }]);
```
Comment (longshuicy) [inline src/components/childComponents/graph/SectorShocksBarChart.jsx:9]: Could you try basing off the latest develop? These changes are not related... :-(

Comment (Rashmil-1999) [reply src/components/childComponents/graph/SectorShocksBarChart.jsx:9]: I did a git pull before :(

**Review by longshuicy — APPROVED** [review]

**Review by Vismayak — APPROVED** [review]
